cloudify_hostpool_plugin/tasks.py

```diff
@@ -0,0 +1,71 @@
+# #######
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+import httplib
+import os
+import requests
+import stat
+
+from cloudify import ctx
+from cloudify.exceptions import NonRecoverableError
+from cloudify.decorators import operation
+
+
+@operation
+def acquire(service_url, **kwargs):
+    ctx.logger.info('Acquire host')
+    response = requests.post('{0}/hosts'.format(service_url))
+    host = response.json()
```

Comment:
you are parsing the host before the verification that the response is ok. what will happen if the request failed? wont this raise an exception?